lib/icons/.icondb.js

```diff
@@ -415,7 +415,7 @@ module.exports = [
 ["config-hs-icon",["medium-purple","dark-purple"],/^haskellconfig\.json$/i,1.5],
 ["config-js-icon",["medium-yellow","dark-yellow"],/\.js(?:beautify|cs|hint)rc$|^(?:jsconfig(?:\..+)?|\.?eshost(?:-config)?)\.json$/i,1.5],
 ["config-perl-icon",["medium-blue","medium-blue"],/^perl[56]?-?config\.json$/i,1.5],
-["config-python-icon",["dark-blue","dark-blue"],/^python-?config\.json$|^pyproject\.toml$|^poetry\.lock$|^\.coveragerc$|^MANIFEST\.in$|\.python-version$|\.flake8$/i,1.5],
+["config-python-icon",["dark-blue","dark-blue"],/^python-?config\.json$|^pyproject\.toml$|^Pipfile?(\.lock)$|^poetry\.lock$|^\.coveragerc$|^MANIFEST\.in$|\.python-version$|\.flake8$/i,1.5],
```

Comment:
What filenames are you attempting to add support for, specifically? The change you've submitted will add support for `Pipfile.lock` and `Pipfil.lock` _(sic)_, but not `Pipfile`.

Is this what you meant?

~~~diff
-^Pipfile?(\.lock)$
+^Pipfile(\.lock)?$
~~~

Reply:
Hey @Alhadis, thanks for taking time out to review this. Yes, there have been a bit of typo I would assume, the `.lock` supposed to be conditionally present.
However, I came across this repo by navigating from this [Chrome Extension](https://chrome.google.com/webstore/detail/file-icons-for-github-and/ficfmibkjjnpogdcfhfokmihanoldbfe?hl=en) and realised it does not support `Pipenv` files as of now. I would be grateful if you could guide me on how I can add support for this!

Reply:
> However, I came across this repo by navigating from this [Chrome Extension](https://chrome.google.com/webstore/detail/file-icons-for-github-and/ficfmibkjjnpogdcfhfokmihanoldbfe?hl=en)

Bloody hell, why am I always the last to find out about projects using my work? 😂 (Even a simple tip-off would be nice…)

Anyway, I can update `config.cson` now, but it might take a while for the changes to propagate downstream (particularly since it's been ages since the last release 😞). Fixing the extension will be fiddly, since it depends on [`file-icons-js`](https://www.npmjs.com/package/file-icons-js), an unofficial port of this package for browser environments. This is precisely why File-Icons is being rehauled (albeit slowly) so there's a single upstream that projects like these can use instead of maintaining disparate forks with spotty coverage. 😢

I fully admit that these are all problems that point back to me being slack at maintaining this project and making progress on things I've openly promised to do… 😥

Reply:
Ah, wait. I see that `Pipenv` is already supported (it has a [dedicated `Pipenv` icon](https://github.com/file-icons/atom/blob/4f7676325dafaca81b4e1ce1939a37d21554388d/config.cson#L5216-L5219)). Is [this](https://github.com/file-icons/icons/blob/76bd3c4f8b5be540b47771b0d53e6a435425c4c2/svg/Pipenv.svg) the icon you're currently seeing for `Pipenv` files?

Reply:
Nope, I am seeing the fallback icon. Pip's `requirements.txt` works just fine on the other hand. Yes, I noticed it's presence in `.cson` file to my surprise, went over that post reading `CONTRIBUTING.md` after I raised this PR to get your attention 😂. Since I am not familiar with repo, I could not fathom why that was not reflected into the artifact that I touched.
If you see this working as expected, I will put my effort onto `file-icons` to keep it updated with the upstream.

PS: I really would love to see the Pipenv icon in my GitHub UI.